index.html

```diff
@@ -14,13 +14,14 @@
     <title>Document</title>
   </head>
   <body>
-    <header class="mx-auto">
-      <h1>
-        <img src="images/logo-small.png" alt="" /><span class="sr-only"
-          >Work out - belief in yourself</span
-        >
+    <header class="navigation mx-auto">
```

Comment:
Why does the header have a classname of `navigation`? Please come up with a better name like `header` or `main-header` or `page-header`.